project/app/Models/Item.php

```diff
@@ -13,6 +13,8 @@ class Item extends Model
     use HasFactory;
     use Notifiable;
 
+    public $timestamps = false;
```

Comment:
Czy mogłabyś napisać po co to było potrzebne? Jakiś błąd się pojawiał?

Reply:
Brakowało kolumny updated at, a to wyłączyło ten błąd, bez jej dodawania